src/unicodeMake.js

```diff
@@ -31,14 +31,19 @@ console.log("export default {");
 
 const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ" +
     "αβγδεϵζηθϑικλμνξοπϖρϱςστυφϕχψωΓΔΘΛΞΠΣΥΦΨΩ";
+const keys = new Set();
 for (const letter of letters) {
     for (const accent of Object.getOwnPropertyNames(accents)) {
         const combined = letter + accent;
         const normalized = combined.normalize('NFC');
         if (normalized.length === 1) {
-            console.log(
-                `    ${encode(normalized)}: ${encode(combined)},`
-                + `  // ${normalized} = ${accents[accent].text}{${letter}}`);
+            const key = encode(normalized);
+            if (!keys.has(key)) {
+                keys.add(key);
+                console.log(
+                    `    ${encode(normalized)}: ${encode(combined)},`
+                    + `  // ${normalized} = ${accents[accent].text}{${letter}}`);
+            }
         }
         for (const accent2 of Object.getOwnPropertyNames(accents)) {
             if (accent === accent2) {
```

Comment:
I wonder if we should remove this `if (accent === accent2) { continue }` block.  Maybe there is an accent that can be doubled into a single character.  (I checked, and there isn't, but maybe better to have the check in the code.)

Reply:
Let's leave it since it isn't hurting anything.